tests/sync.rs

```diff
@@ -1,38 +1,39 @@
-mod sync {
```

Comment:
Nice to get rid of a scope here!

Reply:
:D yeah - figured it already shows which file it's part of, so that's enough scopes! :D